tests/spread/plugins/v2/snaps/make-hello/hello.c

```diff
@@ -1,7 +1,13 @@
 #include <stdio.h>
 
+#if HELLO == 1
```

Comment:
The hello handling could be simplified a bit:

```
#ifndef HELLO
#define HELLO "hello"
#endif

static const char *hello = HELLO;
```
and
```
    make-parameters:
      - HELLO="goodbye"
```
Then you could reuse it for testing rebuild when you detect property changes :D

Reply:
I realize that's not quite wired for cflags, and my head was in cmake mode.

Ignore that.  But perhaps we should have a test case with quotes...